aries-site/src/pages/templates/user-feedback-collection.mdx

```diff
@@ -106,7 +106,7 @@ Click on "Power 1 Device" to see the progressive example of the feedback questio
   height="650px"
   width="100%"
 >
-  <SolicitedFeedbackExample />
+  <ModalTaskFlowExample />
```

Comment:
Code links above should be updated once this merges in with the original branch.

Reply:
done.